CMakeLists.txt

```diff
@@ -10,7 +10,7 @@
 # CMake versions greater than the ITK_NEWEST_VALIDATED_POLICIES_VERSION policies will
 # continue to generate policy warnings "CMake Warning (dev)...Policy CMP0XXX is not set:"
 #
-set(ITK_OLDEST_VALIDATED_POLICIES_VERSION "3.16.3")
+set(ITK_OLDEST_VALIDATED_POLICIES_VERSION "3.18.0")
```

Comment:
Should we instead consider bumping up to 3.22.1, which was [shipped with ubuntu 22.04](https://launchpad.net/ubuntu/jammy/+source/cmake)?

Reply:
I agree with @dzenanz.  It gives us the longest runway without an additional change.

Reply:
+1